app/Section.js

```diff
@@ -9,16 +9,17 @@ const objectPath = require('object-path')
 
 class Section extends React.Component {
   render () {
-    let budgets = (this.props.profile.budgets || []).filter(budget => (this.props.metrics.indexOf(budget.metric) !== -1))
+    const { metrics, lastResult, title, id, yLabel } = this.props;
+    let budgets = (this.props.profile.budgets || []).filter(budget => (metrics.indexOf(budget.metric) !== -1))
```

Comment:
To me, it doesn't look like you're redeclaring this variable? :)
try `const` over `let`